### PR TITLE
add service review fixtures for image and add nav.html

### PR DESCRIPTION
**Pull request by markus0059**

### CODE REVIEW

**Review by MehdiDAAJI — APPROVED** [review]

**Review by Benja-san — CHANGES_REQUESTED** [review]
Waitin for fixtures changes and global paths

File: assets/styles/app.scss [hunk 1]
```diff
@@ -1,3 +1,8 @@
 //start coding by removing this file and adding yours ;)
 @import "wcs";
 @import "~bootstrap/scss/bootstrap";
+
+img {
```
Comment (Benja-san) [inline assets/styles/app.scss:4]: remove this part

File: public/assets/app.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ * (and its CSS file) in your base layout (base.html.twig).
+ */
+
+// any CSS you import will output into a single css file (app.css in this case)
+import "./styles/app.scss";
+
+// start the Stimulus application
+import "./bootstrap";
+
+const $ = require("jquery");
```
Comment (Benja-san) [inline public/assets/app.js:14]: Please do not use Jquery

File: public/assets/controllers/hello_controller.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { Controller } from '@hotwired/stimulus';
```
Comment (Benja-san) [inline public/assets/controllers/hello_controller.js]: remove this demo controller

File: public/assets/styles/_wcs.scss [hunk 4]
```diff
@@ -0,0 +1,40 @@
+@import url('https://fonts.googleapis.com/css2?family=Varela+Round&display=swap');
```
Comment (Benja-san) [inline public/assets/styles/_wcs.scss]: remove useless css template

File: public/assets/styles/app.scss [hunk 5]
```diff
@@ -0,0 +1,8 @@
+//start coding by removing this file and adding yours ;)
+@import "wcs";
```
Comment (Benja-san) [inline public/assets/styles/app.scss:2]: remove wcs import

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Pull request merged**